src/decorator.ts

```diff
@@ -66,13 +66,41 @@ export interface EnumValueMetadata {
 
 export interface FieldOption {
     type?: any;
+    description?: string;
```

Comment:
What about the pagination? It would be nice to have it here as well.

Reply:
Yes I agree, but first I've implemented just description field to get your feedback and pagination can be implemented with the same way later.